tools/respec2html.js

```diff
@@ -30,6 +31,13 @@ if (args.indexOf("-w") !== -1) {
     reportWarnings = true;
 }
 
+if (args.indexOf("--exclude-script") !== -1) {
+    var idx = args.indexOf("--exclude-script");
+    var values = args.splice(idx, 2);
```

Comment:
Are you certain you want "splice" here (instead of `slice`)? Splice modifies the args array, which is not great IMO (i.e., we should keep inputs immutable).